src/login/logind-dbus.c

```diff
@@ -3440,6 +3497,30 @@ static const sd_bus_vtable manager_vtable[] = {
                                               "b", existing),
                                 method_create_session,
                                 0),
+        SD_BUS_METHOD_WITH_ARGS("CreateSessionWithPIDFd",
+                                SD_BUS_ARGS("u", uid,
+                                            "h", pidfd,
+                                            "s", service,
+                                            "s", type,
+                                            "s", class,
+                                            "s", desktop,
+                                            "s", seat_id,
+                                            "u", vtnr,
+                                            "s", tty,
+                                            "s", display,
+                                            "b", remote,
+                                            "s", remote_user,
+                                            "s", remote_host,
+                                            "a(sv)", properties),
```

Comment:
when adding new complex D-Bus methods nowadays I like to always also append a `t flags` argument, even if unused at the beginning. Often enough we find that we want to do slight tweak or optional changes, and it turns out to be useful.